### PR TITLE
Update he.coffee - 2

### PR DESCRIPTION
**Pull request by ChaosCat**
second update done

### CODE REVIEW

File: app/locale/he.coffee [hunk 1]
```diff
@@ -157,43 +157,43 @@ module.exports = nativeDescription: "עברית", englishDescription: "Hebrew",
...
     results: "תוצאות"
     description: "תיאור"
-#    or: "or"
+    or: "או"
     subject: "נושא"
     email: "מייל"
     password: "סיסמה"
     message: "הודעה"
-#    code: "Code"
+    code: "קוד" #I Still need to know if it is a Verb or a Noun
```
Comment (Imperadeiro98) [inline app/locale/he.coffee]: It is a noun.

File: app/locale/he.coffee [hunk 2]
```diff
@@ -237,61 +237,61 @@ module.exports = nativeDescription: "עברית", englishDescription: "Hebrew",
...
+    victory_sign_up: "הירשם על מנת לשמור התקדמות"
+    victory_sign_up_poke: "!רוצה לשמור את הקוד? הירשם בחינם עכשיו"
+    victory_rate_the_level: " :דרג את השלב" # Only in old-style levels.
+    victory_return_to_ladder: "חזור לסולם"
+    victory_play_continue: "המשך"
+    victory_saving_progress: "שומר התקדמות"
+    victory_go_home: "חזור הביתה" # Only in old-style levels.
+    victory_review: "!ספר לנו עוד" # Only in old-style levels.
+    victory_hour_of_code_done: "?סיימת"
+    victory_hour_of_code_done_yes: "Yes, I'm finished with my Hour of Code™!" #I am not sure how to translate it with the [TM] sign
```
Comment (Imperadeiro98) [inline app/locale/he.coffee]: Translate everything minus the Hour of Code and TM.

**Comment by Imperadeiro98** [comment]
Thanks, @ChaosCat!

File: app/locale/he.coffee (hunk 1 above)
Comment (ChaosCat) [inline app/locale/he.coffee]: Ok so I got it right, thanks.

File: app/locale/he.coffee (hunk 2 above)
Comment (ChaosCat) [inline app/locale/he.coffee]: Ok I will write it in the next update.
P.S: Just wondering, how much time does it take until the merged branch is updated on the site itself?

Comment (Imperadeiro98) [inline app/locale/he.coffee]: Not sure. It depends on when the master branch is sent to production.

**Pull request merged**